docs/user_guides/community/privateai.md

```diff
@@ -1,10 +1,10 @@
 # Private AI Integration
 
-[Private AI](https://docs.private-ai.com) allows you to detect and mask Personally Identifiable Information (PII) in your data. This integration enables NeMo Guardrails to use Private AI for PII detection in input, output and retrieval flows.
+[Private AI](http://docs.private-ai.com/?utm_medium=github&utm_campaign=nemo-guardrails) allows you to detect and mask Personally Identifiable Information (PII) in your data. This integration enables NeMo Guardrails to use Private AI for PII detection in input, output and retrieval flows.
```

Comment:
Hey, I noticed that the docs links are HTTP now instead of HTTPS. Shouldn't we use HTTPS?